webcompat/static/js/lib/bugform.js

```diff
@@ -75,13 +75,18 @@ function BugForm() {
       _.bind(this.checkDescriptionValidity, this)
     );
     this.problemType.on("change", _.bind(this.checkProblemTypeValidity, this));
+    this.stepsToReproduceField.on(
+      "focus",
+      _.bind(this.clearStepsToReproducePlaceholder, this)
+    );
     this.uploadField.on("change", _.bind(this.checkImageTypeValidity, this));
     this.osField
       .add(this.browserField)
       .on("blur input", _.bind(this.checkOptionalNonEmpty, this));
     this.submitButtons.on("click", _.bind(this.storeClickedButton, this));
     this.submitButtons.on("click", _.bind(this.loadingIndicator.show, this));
     this.form.on("submit", _.bind(this.maybeUploadImage, this));
+    this.form.on("submit", _.bind(this.clearStepsToReproducePlaceholder, this));
```

Comment:
@miketaylr is this what you meant for me to do? usually I would write one function as the handler for form submit and place related functions within. but it didn't seem right to put clearStepsToReproducePlaceholder() inside maybeUploadImage().